torchtitan/config_manager.py

```diff
@@ -299,7 +299,6 @@ def __init__(self):
         self.parser.add_argument(
             "--experimental.pipeline_parallel_schedule",
             type=str,
-            choices=["1F1B", "Gpipe", "Interleaved1F1B", "FlexibleInterleaved1F1B"],
```

Comment:
May I ask why removing the choices? We probably still want them somewhere to let user know what are accepted schedule (in string formats).